config.cmake

```diff
@@ -203,6 +207,7 @@ if(DEFINED KernelDTSList AND (NOT "${KernelDTSList}" STREQUAL ""))
                 "${device_dest}" --hardware-config "${config_file}" --hardware-schema
                 "${config_schema}" --yaml --yaml-out "${platform_yaml}" --sel4arch
                 "${KernelSel4Arch}" --addrspace-max "${KernelPaddrUserTop}"
+                --json --json-out "${platform_json}"
```

Comment:
It would be nice if we could make CMake aware of the files this step creates, so it can track the dependencies and rebuild things automatically. However, since this seems missing for the YAML also, it's not be a blocker to merge this. Not sure if this can be done trivially anyway, since this does not even look like a CMake target, but runs at CMake's generation step?